Source/WebKit/UIProcess/mac/DisplayLink.cpp

```diff
@@ -86,16 +84,16 @@ WebCore::FramesPerSecond DisplayLink::nominalFramesPerSecondFromDisplayLink(CVDi
     return result ?: WebCore::FullSpeedFramesPerSecond;
 }
 
-void DisplayLink::addObserver(IPC::Connection::UniqueID connectionID, DisplayLinkObserverID observerID, WebCore::FramesPerSecond preferredFramesPerSecond)
+void DisplayLink::addObserver(Client& client, DisplayLinkObserverID observerID, WebCore::FramesPerSecond preferredFramesPerSecond)
 {
     ASSERT(RunLoop::isMain());
 
     LOG_WITH_STREAM(DisplayLink, stream << "[UI ] DisplayLink " << this << " for display display " << m_displayID << " add observer " << observerID << " fps " << preferredFramesPerSecond);
 
     {
         Locker locker { m_observersLock };
-        m_observers.ensure(connectionID, [] {
-            return ConnectionClientInfo { };
+        m_observers.ensure(&client, [] {
+            return ClientInfo { };
         }).iterator->value.observers.append({ observerID, preferredFramesPerSecond });
```

Comment:
I think it will make changes a lot clearer if we rename `m_observers` to `m_clients`.

Reply:
Done.